### ISSUE TITLE
Safari search bar sometimes text appears when typing

### ISSUE THREAD

**Issue by digitalec** [digitalec/thinkmac-t470#1]
Sometimes when doing a search in the address/search bar of Safari, the entire field is "reset". Could be a Safari bug or touchpad issue (palm detection?)

**Comment by rguarascia** [comment]
I didn't want to create another issue for this so i'll ask here.

First of all, thank you so much for constantly updating this repo! It has proven to be a huge help for me to get my HackPad up and running. 

I have everything working besides the battery icon / stats. I am wondering if you could provide a little help if possible. 

I have coped over `SSDT-BATC-T470.aml` to my /patched folder, and added the 4 lines to` ACPI -> DSDT -> Patches.`

I have also made sure to include the `SMCBatteryManager.kext` into my kext / other folder.

Wondering if I have missing anything super obvious here? 

I have the ThinkPad t470 just like you, however the 7th gen i5 variant. 

Thanks again for the repo and replying if you have the time, cheers!

**Comment by digitalec** [comment]
Looks like I left out an important step in the instructions. There is a file located in Extras/DSDT Patches called "Battery Indicator". You need to apply that to your DSDT for it to work. I'll update the README file now but let me know how it works out for you.

**Comment by rguarascia** [comment]
Applied the patch and it worked perfectly! Thank you so much! 

FYI there is a group of people working on bringing Intel WiFi to hacks. I've tried version 0707 and I have WiFi with the AC8260! -> [https://gitter.im/OpenIntelWireless/itlwm](https://gitter.im/OpenIntelWireless/itlwm) the connectivity is a little flakey and the speeds aren't the best but at least they are making progress! 

Cheers man!

**Comment by digitalec** [comment]
Awesome! Glad to hear that was all it was.

Intel wireless support would be huge! I ended up with an airport card and an adapter which worked good but definitely was a tight fit.

**Comment by rguarascia** [comment]
I bet! Also, if you want to update the readme, I have tested SD card reader and it works! Plus headphone jack 😄 

Thanks again!

**Comment by digitalec** [comment]
Excellent! Did you have to do anything to get them to work? I forgot about those lol

**Comment by rguarascia** [comment]
Nope, I followed your guide step by step (besides grabbing the HD620 config as i have the 7th gen i5) and everything seems to been working! 

Only thing that bugs me is the trackpad (it likes to jump around and moving the mouse is slow) but i think that is to be expected!

**Comment by digitalec** [comment]
Yeah, I had that issue with the trackpad as well as I used to use two fingers. I would use my middle finger to move the cursor and then my index finger to press/click which half the time caused it to jump to another portion of the screen and select/open something.

I eventually had to "relearn" the trackpad by using my left thumb (or right middle finger) to click the physical left mouse button while using my right index finger along the trackpad itself. Still not perfect but it definitely makes me miss the touchpad on my old MacBook Pro.
